### DISCUSSION TITLE
General queries on dispatch

### DISCUSSION THREAD

**Discussion by csetarun2** [Netflix/dispatch#2581] [Q&A] [answered]
1. In the docker-compose.yml file , I could see 4 container (web, scheduler, core, postgres), 
  Do we require all these, I have tried with web and scheduler, do we even require both these is dispatch-web container is not sufficient ? what is this dispatch-scheduler do?

2.  If I configure slack plugins info via UI, where this data will be store, will it store in postgres DB ? (since Im using ECS if my container die, and if It will spin up in other ec2 instacne, if its not stored in bd, I have to configure it again )

3. How can we disable cost metric in dispatch application

**Comment by mvilanova** [comment, marked as answer]
1. The core, web, and scheduler should be running on the same container and the postgres one separately. You need all of them. The scheduler is responsible for running the scheduled jobs you can see in the different `scheduled.py` modules for each data type.
2. The data is stored in the postgres database and should be available unless it gets deleted.
3. You can disable it by deleting the default cost type. If there's no default cost type defined, then Dispatch does not calculate incident cost.

**Comment by csetarun2** [comment]
Thanks @mvilanova  for answering my queries

the below docker-compose definition and 1 container is sufficient to run all application code (core,sceduler,web) right ?
![image](https://user-images.githubusercontent.com/21236859/194286046-04bbf289-902c-4c16-b9fb-c0b8c31b8171.png)

**Comment by csetarun2** [comment]
Where can I check any update/code change happened for the dispatch application, Is there any place where we are releasing dispatch versions ?

**Reply by mvilanova** [reply]
We merge code changes to the master branch and then cut at least a new release every month. You can see the list of releases and the more relevant changes they include here: https://github.com/Netflix/dispatch/releases

**Reply by csetarun2** [reply]
Thanks @mvilanova :) for answering 
do we require scheduler separately ?
![image](https://user-images.githubusercontent.com/21236859/195365850-ce0ecae7-8154-470a-96d1-8a0432a06b3d.png)
